src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/TrustDataDao.java

```diff
@@ -16,6 +16,9 @@ public class TrustDataDao {
     @Field("creation_date")
     private LocalDate creationDate;
 
```

Comment:
Needs to be changed to `ceased_date` to be consistent with the web field name. Here and elsewhere.

Reply:
Done.